tasks/orca_catalog_reporting_dummy/schemas/input.json

```diff
@@ -1,41 +1,48 @@
-{
-	"$schema": "http://json-schema.org/draft-07/schema#",
-	"$id": "https://github.com/nasa/cumulus-orca/blob/master/tasks/orca_catalog_reporting_dummy/schemas/input.json",
-	"title": "orca_catalog_reporting_dummy Lambda Input",
-	"description": "The input for the orca_catalog_reporting_dummy Lambda when retrieving reconciliation reports.",
-	"type": "object",
-	"properties": {
-		"providerId": {
-			"description": "The unique ID of the provider making the request.",
-			"type": "array",
-			"items": {
-				"type": "string"
-			}
-		},
-		"collectionId": {
-			"description": "The unique ID of collection(s) to compare.",
-			"type": "array",
-			"items": {
-				"type": "string"
-			}
-		},
-		"granuleId": {
-			"description": "The unique ID of granules to compare.",
-			"type": "array",
-			"items": {
-				"type": "string"
-			}
-		},
-		"startTimestamp": {
-			"description": "Start time for TBD date range to compare data.",
-			"type": "string",
-			"pattern": "^(-?(?:[1-9][0-9]*)?[0-9]{4})-(1[0-2]|0[1-9])-(3[01]|0[1-9]|[12][0-9])T(2[0-3]|[01][0-9]):([0-5][0-9]):([0-5][0-9])(.[0-9]+)?\\+00:00$"
-		},
-		"endTimestamp": {
-			"description": "End time for TBD date range to compare data.",
-			"type": "string",
-			"pattern": "^(-?(?:[1-9][0-9]*)?[0-9]{4})-(1[0-2]|0[1-9])-(3[01]|0[1-9]|[12][0-9])T(2[0-3]|[01][0-9]):([0-5][0-9]):([0-5][0-9])(.[0-9]+)?\\+00:00$"
-		}
-	},
-	"required": []
-}
\ No newline at end of file
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://github.com/nasa/cumulus-orca/blob/master/tasks/catalog_reporting/schemas/input.json",
+    "title": "catalog_reporting Lambda Input",
+    "description": "The input for the catalog_reporting Lambda.",
+    "type": "object",
+    "required": [
+      "pageIndex",
+      "endTimestamp"
+    ],
+    "properties": {
+      "providerId": {
+        "description": "The unique ID of the provider(s) making the request.",
+        "type": "array",
+        "items": {
+          "type": "string"
+        }
+      },
+      "collectionId": {
+        "description": "The unique ID of collection(s) to compare.",
+        "type": "array",
+        "items": {
+          "type": "string"
+        }
+      },
+      "granuleId": {
+        "description": "The unique ID of granule(s) to compare.",
+        "type": "array",
+        "items": {
+          "type": "string"
+        }
+      },
+      "startTimestamp": {
+        "description": "Cumulus createdAt start time for date range to compare data.",
+        "type": "string",
+        "pattern": "^(-?(?:[1-9][0-9]*)?[0-9]{4})-(1[0-2]|0[1-9])-(3[01]|0[1-9]|[12][0-9])T(2[0-3]|[01][0-9]):([0-5][0-9]):([0-5][0-9])(.[0-9]*)?Z"
+      },
+      "endTimestamp": {
+        "description": "Cumulus createdAt end-time for date range to compare data.",
+        "type": "string",
+        "pattern": "^(-?(?:[1-9][0-9]*)?[0-9]{4})-(1[0-2]|0[1-9])-(3[01]|0[1-9]|[12][0-9])T(2[0-3]|[01][0-9]):([0-5][0-9]):([0-5][0-9])(.[0-9]*)?Z"

```

Comment:
I think we decided to use this pattern last time
`^(-?(?:[1-9][0-9]*)?[0-9]{4})-(1[0-2]|0[1-9])-(3[01]|0[1-9]|[12][0-9])T(2[0-3]|[01][0-9]):([0-5][0-9]):([0-5][0-9])(.[0-9]+)?(Z)?\\+00:00$`